pandas/io/pytables.py

```diff
@@ -1644,14 +1643,12 @@ def __init__(
 
         self.auto_close = auto_close
 
-    def __iter__(self):
```

Comment:
Hmm on board with the goal, but is this really what we want to do? Wouldn't this mean that the TableIterator class is actually no longer an iterator by definition?

Reply:
I guess.  In the case that doesn't go through 1674-1675 below TableIterator is already not really an iterator, so in that sense this makes the name _consistently_ inaccuraet

Reply:
Ha sounds good. I'll defer to others more familiar with this code